NashTeamAlpha/BangazonWeb
# Added Product Type Model

## Pull Request Description

Wrote the code to create the columns in the ProductType table
## Steps to Test
1. Go to the ddw-productType-model branch
2. Go to the Models folder and open the ProductType.cs file
3. Check to make sure all properties from the issue ticket are correctly implemented 
## Link to Feature Ticket
#7
